docs/data-sources/cce_cluster_v3.md

```diff
@@ -49,6 +49,10 @@ All above argument parameters can be exported as attribute parameters along with
 
 * `container_network_type` - The container network type: overlay_l2 , underlay_ipvlan or vpc-router.
 
+* `eni_subnet_id` - ENI subnet ID. Specified when creating a CCE Turbo cluster.
+
+* `eni_subnet_cidr` - ENI network segment. Specified when creating a CCE Turbo cluster.
```

Comment:
I suppose, it should be put like "when requesting info for", not "when creating", as it's a data source

Reply:
fixed)